Spec/Auth.swift

```diff
@@ -2441,6 +2441,26 @@ class Auth : QuickSpec {
                     }
                 }
 
+                it("example: if is initialised with custom ttl and a TokenParams object is passed with null, then the @ttl@ used for every subsequent authorization will be null") {
+                    let options = AblyTests.commonAppSetup()
+                    options.defaultTokenParams = { $0.ttl = 0.1; return $0 }(ARTTokenParams())
+                    let rest = ARTRest(options: options)
+
+                    waitUntil(timeout: testTimeout) { done in
+                        rest.auth.authorise(nil, options: nil) { tokenDetails, error in
```

Comment:
Done 6a99e08.

Now the test fails with `<Error Domain=io.ably.cocoa Code=40101 "" UserInfo={ARTErrorStatusCode=401, NSLocalizedDescription=, NSLocalizedFailureReason=Request mac does not match}>`.

The `ttl` spec says that _when the `ttl` is omitted, the REST API default of 60 minutes is applied by Ably_, so should the client set the default value if the `ttl` is missing (ttl=0)  or should the attribute be omitted on the signing method? I'm mean [here](https://github.com/ably/ably-ios/blob/6a99e08d5bba78dcdcbc9242601bd0ad6bda37a7/Source/ARTTokenParams.m#L157).